<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GroupPropWidget</class>
 <widget class="QWidget" name="GroupPropWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>319</width>
    <height>151</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Параметры</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="1">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>188</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="3">
    <widget class="QLineEdit" name="valueUnitEdit">
     <property name="placeholderText">
      <string>Ед. изм.</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1" colspan="2">
    <widget class="QDoubleSpinBox" name="valueSpin">
     <property name="minimum">
      <double>-99999999.000000000000000</double>
     </property>
     <property name="maximum">
      <double>99999999.000000000000000</double>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QCheckBox" name="valueCheck">
     <property name="text">
      <string>значение</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="4">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QPushButton" name="finishBtn">
       <property name="text">
        <string>Завершить редактирование</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="delBtn">
       <property name="maximumSize">
        <size>
         <width>31</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="toolTip">
        <string>Удалить редактируемый объект</string>
       </property>
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="../../../../etc/meteo.icons.qrc">
         <normaloff>:/meteo/icons/delete-16.png</normaloff>:/meteo/icons/delete-16.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>valueSpin</tabstop>
  <tabstop>valueUnitEdit</tabstop>
 </tabstops>
 <resources>
  <include location="../../../../etc/meteo.icons.qrc"/>
 </resources>
 <connections/>
</ui>
